server/main.go

```diff
@@ -45,7 +43,7 @@ func (app *App) setupRoutes() {
 
 	/* GET endpoint that calls app's ___ function defined in maintenance.go */
 	// r.GET("/maintenance_requests", app.AllMaintenanceRequests())
-
+	app.server.POST("/maintenance_request", app.CreateMaintenanceRequests)
```

Comment:
Sorry, this needs to be plural form, haha. `/maintenance_requests`

Reply:
fix: renamed to `/maintenance_requests`